File: Exercícios de Lógica/Python/Exercicio26.py
'''
Exercicio 026
Faça um programa que pergunte o preço de três produtos e informe qual produto você deve comprar, sabendo que a decisão é sempre pelo mais barato.
'''

num1=float(input("Digite o preço do primeiro produto: "))
num2=float(input("Digite o preço do segundo produto: "))
num3=float(input("Digite o preço do terceiro produto: "))

if num1<num2 and num1<num3:
    print("Você deve comprar o primeiro produto")


elif num2<num1 and num2<num3:
    print("Você deve comprar o segundo produto")

else:
    print("Você deve comprar o terceiro produto")

File: Exercícios de Lógica/Java/Extra/Exrcicios_Java/src/Exercicio26.java
/*
Faça um programa que pergunte o preço de três produtos
e informe qual produto você deve comprar, sabendo que a decisão é sempre pelo mais barato.
 */

import java.util.Scanner;
public class Exercicio26 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String produto1 = "banana";
        String produto2 = "melão";
        String produto3 = "melancia";

        System.out.print("Digite o preço da " + produto1 + " :");
        float preco_banana = scanner.nextFloat();

        System.out.print("Digite o preço da " + produto2 + " :");
        float preco_melao = scanner.nextFloat();

        System.out.print("Digite o preço da " + produto3 + " :");
        float preco_melancia = scanner.nextFloat();

        if (preco_banana <= preco_melao && preco_banana <= preco_melancia) {
            System.out.println("Você deve comprar " + produto1);

        } else if (preco_melao <= preco_banana && preco_melao <= preco_melancia) {
            System.out.println("Você deve comprar " + produto2);
        } else {
            System.out.println("Você deve comprar " + produto3);

        }
        scanner.close();

    }
}
